app/src/main/java/com/courseproject/tindar/ds/DatabaseHelper.java

```diff
@@ -414,5 +441,46 @@ public ArrayList<String[]> readMatchList(String userId) {
         cursor.close();
         return matchListResponse;
     }
+
+    @Override
+    public ArrayList<LikeListDsResponseModel> readDisplayNames(ArrayList<String> userIds) {
```

Comment:
I wonder if this logic should happen in a different file. Maybe I'm just biased because this class is already cluttered from every non-chat thing that is database-related being in this class.